WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1670,3 +1670,32 @@ yellowgreen =           #9acd32
 VeilOverlayC1       =   @AltC@
 VeilOverlayC2       =   @darken(AltC,20)@
 VeilOverlayGrad     =   @grad(VeilOverlayC1,VeilOverlayC2,linear-vertical)@
+
+######################################################################
+# SEARCH TOOLBAR 
+######################################################################
+
+SearchResultToolbar             = background-color:@AppC@; @BottomSeparator@; padding: 0 @SkinWrapperPadding@; box-sizing: border-box; 
+SearchResultInputField          = padding: 0 @SkinWrapperPadding@; width:100%; position: relative;
+SearchResultInputFieldIcon      = position: absolute; top: 50%; margin-top: -10px; right: 24px;
+SearchResultText                = @Text-light@ @FontSize-slightly-bigger@ white-space:nowrap; @AlignRight@ vertical-align:middle; width:15%; min-width:200px;
+SearchResultButton              = display: table-cell;
+SearchFilterContainer           = background:@white@; @roundCorners(2px)@; border-color: @PanelBorderColor@;
```

Comment:
I think `InputBgColor` would be more appropriate. 
Updated: https://github.com/Zimbra/zm-web-client/pull/8/files#diff-16ae02866e4f63813de9a7825a1057e3R1683